index.d.ts

```diff
@@ -1381,6 +1390,81 @@ declare namespace Eris {
       RESUMED: 9;
       DISCONNECT: 13;
     };
+    ApplicationCommandTypes: {
```

Comment:
Alphabetical order pls

Reply:
reeee

Reply:
we need a rule for this, current ones don't cover this

Reply:
done